flink-core/src/test/java/org/apache/flink/api/common/typeutils/base/EnumSerializerTest.java

```diff
@@ -27,24 +27,23 @@
 import org.apache.flink.util.InstantiationUtil;
 import org.apache.flink.util.TestLogger;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
-import java.util.Arrays;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
-public class EnumSerializerTest extends TestLogger {
+class EnumSerializerTest extends TestLogger {
```

Comment:
```suggestion
class EnumSerializerTest {
```

Already exists in src/test/resources/META-INF/services/org.junit.jupiter.api.extension.Extension